composer.lock

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
@DivyanshKhanna26 not sure if we need this file at all in repo... I would add it to .gitignore